src/utils/sound_loader.py

```diff
@@ -1,11 +1,33 @@
 from src.conf import config
+import os
+import glob
+import soundfile as sf
+
 import math
 import numpy as np
 import scipy.signal as sig
 
 class SoundLoader:
     def __init__(self):
         # tak się dobiera do ustalonych parametrów (wszystkie są w configurator.py)
+        print(Configurator.parse_analysis()['sampling_frequency'])
+
+
+    def sound_load_file():
+
+        i = 0
+        sound_list = [{}]
+
+        print("Podaj ścieżkę pliku z nagraniami: ")
+        paths = input()
+        for filename in glob.glob(os.path.join(paths, '*.wav')):
+            wav_data, fs = sf.read(filename)
+            sound_list.append({"nazwa": os.listdir(paths)[i],"fs": fs, "wav": wav_data})
```

Comment:
os.listdir(paths)[i] bym zamienił na os.path.splitext(filename)[0] albo skoro operujemy tylko na plikach .wav, można po prostu uciąć ostatnie 4 znaki (.wav) w ten sposób filename[:-4]

Reply:
tylko filename zwraca całą ścieżkę właśnie, od C://......, więc wraz z różną długościa wpliku to mi było ciężej ogarnąć niż taką funkcją

Reply:
żartowałam
działa XD

Reply:
ogarnięte :D